proposals/0002-code-gallery.md

```diff
@@ -0,0 +1,134 @@
+# Proposal: Unity tutorial refactor
```

Comment:
[View in Markdown](https://github.com/clockworklabs/spacetime-docs/pull/27/files?short_path=fc133e8)